src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -16,20 +19,39 @@
 
 public class SearchConceptsQueryBuilder {
 
+  // look up domainID in domainOptions
+
   private SearchConceptsQueryBuilder() {}
 
   public static String buildSearchConceptsQuery(
-      String domainId,
+      SnapshotBuilderDomainOption domainOption,
       String searchText,
       TableNameGenerator tableNameGenerator,
       CloudPlatformWrapper platform) {
     var conceptTablePointer = TablePointer.fromTableName("concept", tableNameGenerator);
+    var occurrenceTable = getOccurrenceTableFromDomain(domainOption.getId());
+    var conditionOccurrencePointer =
```

Comment:
Could conditionOccurrencePointer could be renamed? since it isn't going to represent just the condition occurrence table, but any occurrence table based on the domain

Reply:
And conditionOccurenceTableVariable

Reply:
Thoughts on domainOccurrencePointer?